README.md

```diff
@@ -27,7 +27,7 @@ A wrapper to run [```Pixel.js```](https://github.com/DDMAL/Pixel.js) on top of [
       # Paths to other jobs
     )
     ```
-  - Increase `RODAN_RUNJOB_WORKING_USER_EXPIRY_SECONDS` (line 87) from `15` to `150000`.
+  - Increase `RODAN_RUNJOB_WORKING_USER_EXPIRY_SECONDS` (line 87) from `15` to `15000000`.
```

Comment:
Is this really necessary?

Reply:
Not anymore! I'm moving the Pixel.js job to a separate server